packages/messaging/server/src/rewire.ts

```diff
@@ -5,11 +5,7 @@ if (process.env.TS_NODE_DEV) {
   const originalRequire = Module.prototype.require as (id: string) => any
 
   const rewire = function (this: (id: string) => any, mod: string) {
-    if (
-      mod.startsWith('@botpress/messaging') &&
-      // we don't want to rewire the legacy channels because we get them from npm
-      mod !== '@botpress/messaging-channels-legacy'
-    ) {
+    if (mod.startsWith('@botpress/messaging')) {
```

Comment:
So should we remove the legacy channel dependency too?

Reply:
Already removed that